src/otx/core/engine/train.py

```diff
@@ -108,7 +111,15 @@ def train(
 
     if cfg.train:
         log.info("Starting training!")
-        trainer.fit(model=model, datamodule=datamodule, ckpt_path=cfg.checkpoint)
+        if cfg.resume:
+            trainer.fit(model=model, datamodule=datamodule, ckpt_path=cfg.checkpoint)
+        else:
+            # load weight
+            if cfg.checkpoint is not None:
+                loaded_checkpoint = torch.load(cfg.checkpoint)
+                model.load_state_dict(loaded_checkpoint["state_dict"])
+            # train
+            trainer.fit(model=model, datamodule=datamodule)
```

Comment:
```suggestion
       # load weight
       if cfg.checkpoint is not None:
           loaded_checkpoint = torch.load(cfg.checkpoint)
           model.load_state_dict(loaded_checkpoint["state_dict"])
       # train
       trainer.fit(model=model, datamodule=datamodule)
```

Is it impossible to drop `cfg.resume`?

Reply:
Furthermore, if `model` can handle `cfg.checkpoint` having `meta_info`, do we need this change?

Reply:
Lightening trainer will load other module's hyperparameters such as optimization, lr scheduling when `trainer.fit(model=model, datamodule=datamodule, ckpt_path=cfg.checkpoint)`
This is useful feature when user wants resume, but this could be unwanted behavior those who wants retrain with previous checkpoint